api/src/main/java/io/strimzi/api/kafka/model/KafkaConnectorSpec.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.strimzi.crdgenerator.annotations.Minimum;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Collections.emptyMap;
+
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({"connectCluster", "class", "tasksMax", "topics", "config"})
+@EqualsAndHashCode
+public class KafkaConnectorSpec implements Serializable, UnknownPropertyPreserving {
+    private static final long serialVersionUID = 1L;
+
+    private String className;
+    private Integer tasksMax;
+    private List<String> topics;
+    private List<KafkaConnectorConfig> config;
+    private Map<String, Object> additionalProperties;
+
+    @Description("The Class for the Kafka Connector")
+    @JsonProperty("class")
+    public String getClassName() {
+        return className;
+    }
+
+    @Description("The maximum number of tasks for the Kafka Connector")
+    @Minimum(1)
+    public Integer getTasksMax() {
+        return tasksMax;
+    }
+
+    @Description("A list of topics to use as input for the Kafka Connector")
+    public List<String> getTopics() {
```

Comment:
Do we still think this is really necessary? I know @pamelipluas felt it would be just as good to let the user add it to the `config` for sink connectors.

Reply:
We can remove it, it's better for consistency with other connectors that do not have this.